README.md

```diff
@@ -47,9 +47,73 @@ goreleaser release --snapshot --clean
 Built binaries will be located in the `dist/` directory.
 
 ## Testing
-SMD includes continuous test (CT) verification for live HPC systems. The test image is invoked via `helm test`.
-In addition to the service itself, this repository builds and publishes cray-smd-test images containing tests that verify HSM on live Shasta systems. The tests are invoked via helm test as part of the Continuous Test (CT) framework during CSM installs and upgrades. The version of the cray-smd-test image (vX.Y.Z) should match the version of the cray-smd image being tested, both of which are specified in the helm chart for the service.
 
+### Running the CT Tests in a Docker Compose Environment
+
+1. Start services using the quick start guide
```

Comment:
I am fine with saying using quick start guide but since we aren't trying to support it as much I'm not sure how much I'd want to put it into docs... Might just be better to say "start services" and leave it agnostic?